actioncable/Rakefile

```diff
@@ -34,7 +34,7 @@ end
 
 namespace :assets do
   root_path = Pathname.new(dir)
-  destination_path = root_path.join("lib/assets/compiled")
+  destination_paths = [ root_path.join("lib/assets/compiled"), root_path.join("dist") ]
```

Comment:
Why two destination paths? Can we point package.json to `lib/assets/compiled/*` instead?